### ISSUE TITLE
if I want to use customed dataset, which config file can i refer to?

### ISSUE THREAD

**Issue by lynshwoo2022** [SanyeungWang/PML#4]
hi there,
thanks for your great work! 
I'm a little confused about those config files for different datasets, especially the `margin` parameter in it. If I want to train this model all from start, should I set it to True? what about other parameters like learning rate?

**Comment by SanyeungWang** [comment]
For a new dataset, you also need to create a new config file to control the parameters more conveniently, otherwise you have to modify it in option.py every time you train or test.

Regardless of whether the value of margin is True or False, this will directly affect line 343 of model/resnet.py. Setting it to True means that m_p of formula 4 in the paper is not 0. 

You should of course set this to True if you want to train the model from the start, otherwise it will become a vanilla method.

For other parameters, you can try to fine-tune them according to the dataset you are using.

If you have further questions, please open an issue.

**Comment by lynshwoo2022** [comment]
thanks
